docs/How the library works/field-library.md

```diff
@@ -1,155 +0,0 @@
-# Field library
-
-We have many common field and widgets that are available for use in forms. You should stick with these common fields and definitions if at all possible.
-
-There are generally two forms of common definitions: schema/uiSchema objects and functions that return schema/uiSchema objects. For the function versions, there will be documentation in the fields for the parameters.
-
-## Widgets
```

Comment:
This went to https://github.com/usds/us-forms-system/pull/112/files#diff-88f3e18582f4e60f773836345bdf7ff7R10.